apps/www/data/Pricing.json

```diff
@@ -262,92 +284,124 @@
       }
     ]
   },
-  "support": {
-    "title": "Support and Compliance",
-    "icon": "M18.364 5.636l-3.536 3.536m0 5.656l3.536 3.536M9.172 9.172L5.636 5.636m3.536 9.192l-3.536 3.536M21 12a9 9 0 11-18 0 9 9 0 0118 0zm-5 0a4 4 0 11-8 0 4 4 0 018 0z",
+  "security": {
+    "title": "Platform Security and Compliance",
+    "icon": "M9 12.75L11.25 15 15 9.75m-3-7.036A11.959 11.959 0 013.598 6 11.99 11.99 0 003 9.749c0 5.592 3.824 10.29 9 11.623 5.176-1.332 9-6.03 9-11.622 0-1.31-.21-2.571-.598-3.751h-.152c-3.196 0-6.1-1.248-8.25-3.285z",
     "features": [
       {
-        "title": "Community support",
+        "title": "On Premises / BYO cloud",
         "tiers": {
-          "free": true,
-          "pro": true,
+          "free": false,
+          "pro": false,
           "enterprise": true
         }
       },
       {
-        "title": "Email support",
+        "title": "Log retention (API & Database)",
+        "tiers": {
+          "free": "1 day",
+          "pro": "7 days",
+          "enterprise": "90 days"
+        }
+      },
+      {
+        "title": "Log drain",
+        "tiers": {
+          "free": false,
+          "pro": false,
+          "enterprise": "Coming soon"
+        }
+      },
+      {
+        "title": "Metrics endpoint",
         "tiers": {
           "free": false,
           "pro": true,
           "enterprise": true
         }
       },
       {
-        "title": "Email support SLA",
+        "title": "SOC2",
         "tiers": {
           "free": false,
           "pro": false,
-          "enterprise": true
+          "enterprise": "Coming soon"
         }
       },
       {
-        "title": "Designated support",
+        "title": "SSO",
         "tiers": {
           "free": false,
           "pro": false,
-          "enterprise": true
+          "enterprise": "Contact Us"
         }
       },
       {
-        "title": "On Premises / BYO cloud",
+        "title": "99.9% SLA",
         "tiers": {
           "free": false,
           "pro": false,
           "enterprise": true
         }
       },
       {
-        "title": "Log retention (API & Database)",
+        "title": "Access Roles",
         "tiers": {
-          "free": "1 day",
-          "pro": "7 days",
-          "enterprise": "90 days"
+          "free": "Owner, Developer",
+          "pro": "Owner, Developer",
+          "enterprise": "Additional owner(s), admin, read-only, billing admin, custom"
         }
       },
       {
-        "title": "Log drain",
+        "title": "Vanity URLs",
+        "tooltip": "We currently screen vanity URLs to ensure they are non-offensive. Please email growth@supabase.io with your project details to request your vanity URL. We process requests in batches once every 14 days.",
```

Comment:
can probably drop 'We currently screen vanity URLs to ensure they are non-offensive.'

Reply:
done